lib/peep.rb

```diff
@@ -0,0 +1,18 @@
+require "pg"
+
+class Peep
+  def self.all
+    ENV["ENVIRONMENT"] == "test" ?
```

Comment:
You could write this instead as

`connection = PG.connect(dbname: ENV["ENVIRONEMENT"] == "test" ? "chitter_test" : "chitter")`

can't decide which is cleaner

Reply:
Or even store the database connection in a separate file, and then require it in! (this is the Interface Segregation Principle)